images/oc-build-deploy-dind/build-deploy-docker-compose.sh

```diff
@@ -694,6 +706,7 @@ fi
 ##############################################
 URL_COUNTER=0
 while [ -n "$(cat .lagoon.yml | shyaml get-value environments.${BRANCH//./\\.}.monitoring_urls.$URL_COUNTER 2> /dev/null)" ]; do
```

Comment:
let's add an `@DEPRECATED` as comment here, so we know that we need to remove this with Lagoon 2.0.0